docs/development/contributing_guide.md

```diff
@@ -148,16 +154,14 @@ The unit tests run parts of Synapse, including your changes, to see if anything
 was broken. They are slower than the linters but will typically catch more errors.
 
 ```sh
-source ./env/bin/activate
-trial tests
+poetry run trial tests
```

Comment:
Does poetry not have a concept of activating the virtualenv?

Reply:
The closest thing is `poetry shell`, but that spawns a new shell rather than modifying the environment in your existing shell.

Reply:
I guess you can still just manually activate the virtualenv in the same way, if you wanted to.

Reply:
Should we include a cross-reference to the direnv bits somewhere? (Or mention `poetry shell` somewhere?)